json-schema/schema.json

```diff
@@ -61,7 +61,6 @@
         { "required": ["s2:generation_time"] },
         { "required": ["s2:processing_baseline"] },
         { "required": ["s2:water_percentage"] },
-        { "required": ["s2:snow_ice_percentage"] },
```

Comment:
elsewhere it says deprecated, but it was removed here.

Reply:
I felt like people should provide at least one field that's not deprecated.